crates/sncast/tests/e2e/account/create.rs

```diff
@@ -242,14 +247,18 @@ pub async fn test_happy_case_keystore() {
     ];
 
     let snapbox = runner(&args).current_dir(temp_dir.path());
+    let output = snapbox.assert().success();
 
-    snapbox.assert().stdout_matches(indoc! {r"
+    assert_stdout_contains(
+        output,
+        indoc! {r"
         command: account create
         add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
         address: 0x[..]
         max_fee: [..]
         message: Account successfully created[..]
-    "});
+        "},
+    );
```

Comment:
I'm not sure whether we want to use `assert_stdout_contains` for consistency (same logic that is applied to `stderr`) in these cases, or keep `stdout_matches`, since we don't expect any stack trace and hereby can strictly match a snippet.